met-web/src/components/imageUpload/index.tsx

```diff
@@ -90,7 +90,7 @@ const ImageUpload = ({
                         }}
                     >
                         <input {...getInputProps()} multiple={false} accept={'image/*'} />
-                        <Typography m={4}>{helpText}</Typography>
+                        <Typography m={2}>{helpText}</Typography>
```

Comment:
This also affect the banner image upload. Just make sure that one doesn't look weird now.
 maybe we can pass this margin as a prop

Reply:
I checked both it looks great on both but im open to that as well

Reply:
done